docs/src/modules/spring/pages/event-sourced-entities.adoc

```diff
@@ -60,11 +60,13 @@ Let's have a look at what our shopping cart entity will look like for the first
 include::example$spring-eventsourced-shopping-cart/src/main/java/com/example/shoppingcart/ShoppingCartEntity.java[tag=class]
 ----
 <1> Create a class that extends `EventSourcedEntity<S>`, where `S` is the state type this entity will store (i.e. `ShoppingCart`).
-<2> Make sure to annotate such class with `@Entity` and pass in an `entityKey` and an `entityType`.
-<3> Make use of Spring's RequestMapping annotation to define the routes to your entity.
+<2> Make sure to annotate such class with `@EntityKey`  and pass name of the path param that will be used as the
```

Comment:
```suggestion
<2> Annotate such class with `@EntityKey`  and pass the name of the key that will be used as the
```

Reply:
I would change other occurrences of `Make sure` to the use of the plain verb. Of course, just a suggestion.